client/app/components/packages/landuse-form/change-city-map.hbs

```diff
@@ -0,0 +1,367 @@
+          onClick={{fn (mut form.data.dcpChangestreetwidthopt)
+            (if form.data.dcpChangestreetwidthopt
+              (optionset 'landuseForm' 'dcpChangestreetwidthopt' 'code' 'No')
+              (optionset 'landuseForm' 'dcpChangestreetwidthopt' 'code' 'Yes')
+            )
+          }}
+        />
+        <label for="dcpChangestreetwidthopt">
+          Street width
+        </label>
+
+        <br>
+
+        <Input
+          type="checkbox"
+          id="dcpChangestreetalignmentopt"
+          data-test-checkbox="dcpChangestreetalignmentopt"
+          checked={{if
+            (eq
+              form.data.dcpChangestreetalignmentopt
+              (optionset 'landuseForm' 'dcpChangestreetalignmentopt' 'code' 'Yes')
+            ) "checked"}}
+          onClick={{fn (mut form.data.dcpChangestreetalignmentopt)
+            (if form.data.dcpChangestreetalignmentopt
+              (optionset 'landuseForm' 'dcpChangestreetalignmentopt' 'code' 'No')
+              (optionset 'landuseForm' 'dcpChangestreetalignmentopt' 'code' 'Yes')
+            )
+          }}
+        />
+        <label for="dcpChangestreetalignmentopt">
+          Street alignment
+        </label>
+
+        <br>
+
+        <Input
+          type="checkbox"
+          id="dcpChangestreetgradeopt"
+          data-test-checkbox="dcpChangestreetgradeopt"
+          checked={{if
+            (eq
+              form.data.dcpChangestreetgradeopt
+              (optionset 'landuseForm' 'dcpChangestreetgradeopt' 'code' 'Yes')
+            ) "checked"}}
+          onClick={{fn (mut form.data.dcpChangestreetgradeopt)
+            (if form.data.dcpChangestreetgradeopt
+              (optionset 'landuseForm' 'dcpChangestreetgradeopt' 'code' 'No')
+              (optionset 'landuseForm' 'dcpChangestreetgradeopt' 'code' 'Yes')
+            )
+          }}
+        />
+        <label for="dcpChangestreetgradeopt">
+          Street grade
+        </label>
+
+        <br>
+
+        <form.Field
+          @attribute="dcpEasement3"
+          @type="checkbox"
+          as |Checkbox|
+        >
+          <Checkbox>
+            Easement
+          </Checkbox>
+        </form.Field>
+
+      </Ui::Question>
+
+      <Ui::Question as |Q|>
+        <Q.Legend>
+          Is there a related acquisition of property?
+        </Q.Legend>
+
+        <form.Field
+          @attribute="dcpRelatedacquisitionofpropertyopt"
+          @type="radio-group"
+          as |RadioGroup|
+        >
+          <RadioGroup
+            @options={{optionset 'landuseForm' 'dcpRelatedacquisitionofpropertyopt' 'list'}}
+          />
+        </form.Field>
+      </Ui::Question>
+
+      <Ui::Question
+        class="no-margin-top"
+        data-test-related-acquisition-radio-group
+        as |Q|
+      >
+        <Q.Legend>
+          Is there a related disposition of property?
+        </Q.Legend>
+
+          <form.Field
+            @attribute="dcpRelatedacquisition"
+            @type="radio"
```

Comment:
Just curious -- can you use the radio-group type here instead?

Reply:
I've seen other examples of boolean radio buttons where we do it this way, not sure what the reason is though, maybe just outdated code -- if there's not a particular reason for doing it this way, I can create another PR fixing up all these cases.